docs/reference/commandline/attach.md

```diff
@@ -14,6 +14,7 @@ parent = "smn_cli"
 
     Attach to a running container
 
+      --detach-keys       Specify the escape key sequence used to detach a container
```

Comment:
@moxiegirl I think the text should be more explicit that the sequence is a comma-separated list. E.g. `ctrl-a,e’, `ctrl-a,ctrl-b`, `ctrl-a,b,c,ctrl-d` are all valid (I think). The first keystroke doesn't have to be `ctrl-...` (although using a regular character is usually impractical)